app/views/layouts/_topbar.html.haml

```diff
@@ -1,5 +1,5 @@
 %div{ class: "top-bar" }
-  %div{ class: "top-bar-left menu-text"}= link_to 'Catechumen', root_path
+  %div{ class: "top-bar-left menu-text"}= link_to 'Catechumen', students_path
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should end with one space before the closing brace